schema/tables/menu_bar.yml

```diff
@@ -0,0 +1,12 @@
+name: menu_bar
```

Comment:
Looks like `schema/tables/` contains osquery tables only:

Should we start adding fleetd tables there too? Or maybe we should add a separate directory for fleetd tables.

@zwass

Reply:
chiming in because I have been looking at the schema to fix some docs. For what is worth, we do have tables from the macadmins extension documented in `schema/tables`, they all have a note like this:

> notes: Requires [macadmins-extension](https://github.com/macadmins/osquery-extension/), which is included by default on osquery packages built with [fleetctl](https://fleetdm.com/docs/using-fleet/adding-hosts#osquery-installer).

Reply:
I was about to check this and hit #9397. Not urgent, but something worth fixing.